### DISCUSSION TITLE
How can you include a ChatbotCard within a Dialog box?

### DISCUSSION THREAD

**Discussion by nivzx** [h2oai/wave#2305] [Q&A]
Hello!
In an app I'm working on, I need to add a Chatbot within a Dialog box. Is it possible to add a ChatbotCard within a `ui.Dialog`?

**Comment by shamilh2o** [comment]
cc: @mturoci

**Comment by mturoci** [comment]
Hey @nivzx. Not possible at the moment, but should be easy to add. Please file a feature request.

**Reply by nivzx** [reply]
Hi, thanks @mturoci. I did file a feature request [here](https://github.com/h2oai/wave/issues/2308)

**Reply by mturoci** [reply]
Perfect!
